www/base/src/app/about/about.tpl.jade

```diff
@@ -5,15 +5,14 @@
         .row
           .col-sm-4
             ul
-              li Buildbot version: {{config.bbversion}}
-              li Twisted version: {{config.txversion}}
               li Project Infos:
-                a(ng-href="{{config.projecturl}}") {{config.projectname}}
+                a(ng-href="{{config.titleURL}}") {{config.title}}
+              li(ng-repeat="v in config.versions" ng-bind-template="{{v[0]}} version: {{v[1]}}")
```

Comment:
@tardyp `ng-bind-template` has an advantage over using template directly inner the HTML tag. The tag will shows nothing before the rendering finished.